node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -58,39 +58,27 @@
     }
 }
 
-/* MediaCore preview */
-
-#filepreview-container #filepreview-mediacore,
+#filepreview-container #filepreview-video-container,
 #filepreview-container #filepreview-markdown-container {
     padding: 20px 0;
 }
 
-#filepreview-container #filepreview-mediacore iframe {
-    width: 100%;
-}
-
-.mobile #filepreview-container #filepreview-mediacore iframe {
-    width: 340px; /* iOS will need an explicit width for the iframe */
-}
-
-#filepreview-container #filepreview-mediacore.filepreview-mediacore-audio iframe {
-    height: 24px;
-}
+/* Video preview */
```

Comment:
It actually does that by default (the background is black), I'll change it back.